docs/design.md

```diff
@@ -38,13 +58,164 @@ spec:
 ```
 
 ### Command
+
 Optionally you can override the default command of a container by providing
 `command` in the Substratus resource:
 ```yaml
 spec:
   command: ["train.sh"]
 ```
 
+### User
+
+All containers should use the `ml` user for all training, importing, notebook, etc operations. The `/home/ml` directory should correspond to the `WORKDIR` of all Dockerfiles.
+
+Non-root users should be enforced using the `runAsNonRoot: true` field within Pod security contexts.
```

Comment:
I do think the usability of Substratus is impacted for folks that need to run custom images. For some folks writing a Dockerfile is hard enough when using root and having to use a specific user might make it too hard for them.

I am not entirely sold that the security benefits outweigh the UX downgrade here.

Reply:
Removed requirement for root.